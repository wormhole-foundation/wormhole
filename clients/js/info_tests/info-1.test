evm info -c Bsc -n mainnet -m TokenBridge
