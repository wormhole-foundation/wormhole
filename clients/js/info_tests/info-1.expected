{
  "address": "0xB6F6D86a8f9879A9c87f643768d9efc38c1Da6E7",
  "wormhole": "0x98f3c9e6E3fAce36bAAd05FE09d375Ef1464288B",
  "implementation": "0x621199f6beB2ba6fbD962E8A52A320EA4F6D4aA3",
  "isInitialized": true,
  "tokenImplementation": "0x7f8C5e730121657E17E452c5a1bA3fA1eF96f22a",
  "chainId": 4,
  "finality": 15,
  "evmChainId": "56",
  "isFork": false,
  "governanceChainId": 1,
  "governanceContract": "0x0000000000000000000000000000000000000000000000000000000000000004",
  "WETH": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
  "registrations": {
    "Solana": "0xec7372995d5cc8732397fb0ad35c0121e0eaa90d26f828a534cab54391b3a4f5",
    "Ethereum": "0x0000000000000000000000003ee18b2214aff97000d974cf647e7c347e8fa585",
    "Terra": "0x0000000000000000000000007cf7b764e38a0a5e967972c1df77d432510564e2",
    "Polygon": "0x0000000000000000000000005a58505a96d1dbf8df91cb21b54419fc36e93fde",
    "Avalanche": "0x0000000000000000000000000e082f06ff657d94310cb8ce8b0d9a04541d8052",
    "Oasis": "0x0000000000000000000000005848c791e09901b40a9ef749f2a6735b418d7564",
    "Algorand": "0x67e93fa6c8ac5c819990aa7340c0c16b508abb1178be9b30d024b8ac25193d45",
    "Aurora": "0x00000000000000000000000051b5123a7b0f9b2ba265f9c4c8de7d78d52f510f",
    "Fantom": "0x0000000000000000000000007c9fc5741288cdfdd83ceb07f3ea7e22618d79d2",
    "Karura": "0x000000000000000000000000ae9d7fe007b3327aa64a32824aaac52c42a6e624",
    "Acala": "0x000000000000000000000000ae9d7fe007b3327aa64a32824aaac52c42a6e624",
    "Klaytn": "0x0000000000000000000000005b08ac39eaed75c0439fc750d9fe7e1f9dd0193f",
    "Celo": "0x000000000000000000000000796dff6d74f3e27060b71255fe517bfb23c93eed",
    "Near": "0x148410499d3fcda4dcfd68a1ebfcdddda16ab28326448d4aae4d2f0465cdfcb7",
    "Moonbeam": "0x000000000000000000000000b1731c586ca89a23809861c6103f0b96b3f57d92",
    "Neon": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Terra2": "0xa463ad028fb79679cfc8ce1efba35ac0e77b35080a1abe9bebe83461f176b0a3",
    "Injective": "0x00000000000000000000000045dbea4617971d93188eda21530bc6503d153313",
    "Osmosis": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Sui": "0xccceeb29348f71bdd22ffef43a2a19c1f5b5e17c5cca5411529120182672ade5",
    "Aptos": "0x0000000000000000000000000000000000000000000000000000000000000001",
    "Arbitrum": "0x0000000000000000000000000b2402144bb366a632d14b83f244d2e0e21bd39c",
    "Optimism": "0x0000000000000000000000001d68124e65fafc907325e3edbf8c4d84499daa8b",
    "Gnosis": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Pythnet": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Xpla": "0x8f9cf727175353b17a5f574270e370776123d90fd74956ae4277962b4fdee24c",
    "Btc": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Base": "0x0000000000000000000000008d2de8d2f73f1f4cab472ac9a881c9b123c79627",
    "Sei": "0x86c5fd957e2db8389553e1728f9c27964b22a8154091ccba54d75f4b10c61f5e",
    "Rootstock": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Scroll": "0x00000000000000000000000024850c6f61c438823f01b7a3bf2b89b72174fa9d",
    "Mantle": "0x00000000000000000000000024850c6f61c438823f01b7a3bf2b89b72174fa9d",
    "Blast": "0x00000000000000000000000024850c6f61c438823f01b7a3bf2b89b72174fa9d",
    "Xlayer": "0x0000000000000000000000005537857664b0f9efe38c9f320f75fef23234d904",
    "Linea": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Berachain": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Seievm": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Wormchain": "0xaeb534c45c3049d380b9d9b966f9895f53abd4301bfaff407fa09dea8ae7a924",
    "Cosmoshub": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Evmos": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Kujira": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Neutron": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Celestia": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Stargaze": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Seda": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Dymension": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Provenance": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Sepolia": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "ArbitrumSepolia": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "BaseSepolia": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "OptimismSepolia": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "Holesky": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "PolygonSepolia": "0x0000000000000000000000000000000000000000000000000000000000000000"
  }
}
